applications/tari_dan_wallet_daemon/src/jrpc_server.rs

```diff
@@ -46,26 +46,17 @@ pub async fn listen(
 
 async fn handler(Extension(context): Extension<Arc<HandlerContext>>, value: JsonRpcExtractor) -> JrpcResult {
     info!(target: LOG_TARGET, "🌐 JSON-RPC request: {}", value.method);
-    if value.method == "rpc.discover" {
-        return Ok(JsonRpcResponse::success(
-            value.id,
-            serde_json::from_str::<HashMap<String, Value>>(include_str!("../openrpc.json")).map_err(|e| {
-                JsonRpcResponse::error(
-                    value.id,
-                    JsonRpcError::new(JsonRpcErrorReason::InternalError, e.to_string(), json!({})),
-                )
-            })?,
-        ));
-    }
+
     match value.method.as_str().split_once('.') {
+        Some(("rpc", "discover")) => call_handler(context, value, rpc::handle_discover).await,
         Some(("keys", method)) => match method {
             "create" => call_handler(context, value, keys::handle_create).await,
             "list" => call_handler(context, value, keys::handle_list).await,
             "set_active" => call_handler(context, value, keys::handle_set_active).await,
             _ => Ok(value.method_not_found(&value.method)),
         },
         Some(("transactions", method)) => match method {
-            "submit" | "claim_burn" => call_handler(context, value, transaction::handle_submit).await,
```

Comment:
Why are we removing the "claim_burn" here ?

Reply:
This method is for submitting a transaction